src/deploy/functions/tasks.ts

```diff
@@ -95,6 +95,14 @@ export function createFunctionTask(
       op = await gcf.createFunction(apiFunction);
     } else {
       const apiFunction = gcfV2.functionFromSpec(fn, params.storageSource!);
+      // N.B. As of GCFv2 private preview GCF no longer creates Pub/Sub topics
+      // for Pub/Sub event handlers. This may change, at which point this code
+      // could be deleted.
+      if (apiFunction.eventTrigger?.pubsubTopic) {
+        await pubsub.createTopic({
```

Comment:
Is this idempotent? What if the topic already exists?

Do we need to ensure the API is enabled first? Or is it going to be automatically since GCF depends on Pub/Sub?

Reply:
Enablement checks add latency (even if small), so if we know it'll be there I think it's fine. Do fix idempotency tho - are you doing that in this PR or a followup?

Reply:
Already fixed

Reply:
Whoops. I never published that part of the change. Fixing.